test/unit/org/apache/cassandra/guardrails/GuardrailDiskUsageTest.java

```diff
@@ -336,8 +336,11 @@ public void testWriteRequests() throws Throwable
             executeNet(batchStatement);
         };
 
+        // delay needed after grant permissions
```

Comment:
perhaps we should have some generic method in `CqlTester` to await for the permission grants/revokes to be applies as it seems to be needed in more tests

Reply:
In STAR-654 I have used `Awaitility`, which is itself generic and flexible. I'm not sure adding a wrapper method would provide additional value? Maybe if the wait until condition is identical in multiple places?

Reply:
I was particularly thinking about modifying permissions since this can be a common problem. Anyway, it is very nit, I'm not going to be picky about this :)

Reply:
Ok, then I will leave this and the change in STAR-654 as they are for now.